apps/demo-diagnostic-report-viewer-smart-app/README.md

```
@@ -0,0 +1,155 @@
+
+The app runs at **http://localhost:3000**.
+
+---
+
+## Launch Flows
+
+### EHR Launch (standard)
+
+The EHR app opens the smart app with launch parameters:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)